apps/dg/resources/dg.css

```diff
@@ -1214,19 +1214,23 @@ input.dg-editor-text {
     background-color: white;
     width: 30px;
 }
-.dg-table-drop-target  .sc-label-view {
-    -webkit-writing-mode: vertical-lr;
-    writing-mode: vertical-lr;
+
+.dg-table-drop-target .sc-label-view {
+    -webkit-writing-mode: vertical-rl;
+    writing-mode: vertical-rl;
     text-align: center;
-    padding: 0.5em;
     transform: rotate(180deg);
+    display: flex;
+    align-items: center;
+    justify-content: center;
 }
+
```

Comment:
Taka, I am curious if you verified the vertical-rl writing mode in Chinese. That translation may be affected by the change.